app/templates/join.hbs

```diff
@@ -1,2 +1,14 @@
 {{page-title 'Join'}}
-<Stepper />
\ No newline at end of file
+{{#if this.login.isLoading}}
+<div data-test-loading class='intro__loading'>
```

Comment:
login button page in component signup-steps/step-zero
like this step-one page for firstname,lastname,username- 
https://github.com/Real-Dev-Squad/website-www/pull/607/files#diff-a04f9bf1d30ebf7d780a7ccd09c47c973d988ed678833dea68f1e1b31c4352c9R1

Reply:
Talked with @rohan09-raj said its fine